packages/core/datadog-configuration.schema.json

```diff
@@ -0,0 +1,190 @@
+                    "type": "string"
+                },
+                "nativeCrashReportEnabled": {
+                    "description": "Enables crash reporting for native platforms (iOS, Android). Default `false`.",
+                    "type": "boolean"
+                },
+                "nativeLongTaskThresholdMs": {
+                    "description": "The threshold for native long tasks reporting in milliseconds.",
+                    "type": "integer"
+                },
+                "longTaskThresholdMs": {
+                    "description": "The threshold for javascript long tasks reporting in milliseconds.",
+                    "type": "integer"
+                },
+                "sessionSamplingRate": {
+                    "description": "Percentage of sampled RUM sessions. Range `0`-`100`.",
+                    "type": "integer"
+                },
+                "site": {
+                    "description": "The Datadog site of your organization (can be 'US1', 'US1_FED', 'US3', 'US5', or 'EU1', default is 'US1').",
+                    "type": "string"
+                },
+                "trackingConsent": {
+                    "description": "Consent, which can take one of the following values: 'PENDING', 'GRANTED', 'NOT_GRANTED'.",
+                    "type": "string",
+                    "enum": ["PENDING", "GRANTED", "NOT_GRANTED"]
+                },
+                "telemetrySampleRate": {
+                    "description": "The sampling rate for Internal Telemetry (info related to the work of the SDK internals).",
+                    "type": "integer"
+                },
+                "vitalsUpdateFrequency": {
+                    "description": "Sets the preferred frequency for collecting mobile vitals.",
+                    "type": "string",
+                    "enum": ["NEVER", "RARE", "AVERAGE", "FREQUENT"]
+                },
+                "uploadFrequency": {
+                    "description": "Sets the preferred frequency for uploading batches of data.",
+                    "type": "string",
+                    "enum": ["RARE", "AVERAGE", "FREQUENT"]
+                },
+                "batchSize": {
+                    "description": "Defines the Datadog SDK policy when batching data together before uploading it to Datadog servers.",
+                    "type": "string",
+                    "enum": ["SMALL", "MEDIUM", "LARGE"]
+                },
+                "trackFrustrations": {
+                    "description": "Enables tracking of frustration signals (error taps). Defaults to `true`.",
+                    "type": "boolean"
+                },
+                "trackBackgroundEvents": {
+                    "description": "Enables tracking of RUM event when no RUM View is active.",
+                    "type": "boolean"
+                },
+                "customEndpoints": {
+                    "type": "object",
```

Comment:
no description

Reply:
This feature isn't officially documented so I'm not sure we should add a description here?